core/store/src/trie/outgoing_metadata.rs

```diff
@@ -585,7 +623,7 @@ mod tests {
         pub fn update_on_receipt_popped(&mut self, receipt_size: ByteSize, receipt_gas: Gas) {
             let mut first_group = self.groups.pop_front().unwrap();
             first_group.total_size -= receipt_size.as_u64() as u128;
-            first_group.total_gas -= receipt_gas as u128;
+            first_group.total_gas -= Into::<u128>::into(receipt_gas.as_gas());
```

Comment:
If `.into()` is ambiguous, I recommend using `u128::from(...)` instead of `Into::<u128>::into(...)`